### DISCUSSION TITLE
Custom page templates

### DISCUSSION THREAD

**Discussion by ctuxboy** [BoidCMS/BoidCMS#16] [Ideas]
Adding the option, when create a post/page, choosing a custom template.

**Comment by sysadev** [comment]
There's also another idea, [Page instead of post as front page](https://github.com/BoidCMS/BoidCMS/discussions/14), which will be added in `v1.1.0` release.

Regarding your idea, there's already a way to distinguish between posts and static pages by choosing the right **type**.
But if you think that giving a specific page a custom template will be of use, then it'll surely be added.

And thanks for the idea.

**Reply by ctuxboy** [reply]
@shoaiyb thanks for your fast answer. Yes i know about the types page and post, but sometimes if you have post, pages and article-posts in one project, it would be handy selecting your own custom template for, by example **article.php** .
Btw nice project. Good work!

**Reply by BoidCMS** [reply]
Thanks for the idea @ctuxboy, it'll be added in `v1.1.0`, which is to be released on September.

**Reply by ctuxboy** [reply]
Nice :smiley: , now i follow your project, will tested very soon.
